samples/webview-samples/multi-tab-browser/main.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Listens for the app launching then creates the window
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ * @see http://developer.chrome.com/apps/app.window.html
+ */
+window.chrome.app.runtime.onLaunched.addListener(function() {
```

Comment:
Changed "window.chrome." to "chrome." in all the apps in "/webview-samples/" folder.